src/py42/clients/detectionlists/departing_employee.py

```diff
@@ -142,121 +120,27 @@ def toggle_alerts(self, tenant_id=None, alerts_enabled=True):
         Returns:
             :class:`py42.sdk.response.Py42Response`
         """
-        tenant_id = tenant_id if tenant_id else self._user_context.get_current_tenant_id()
-        uri = self._uri_prefix.format(u"togglealerts")
+        tenant_id = self._user_context.get_current_tenant_id()
+        uri = self._uri_prefix.format(u"setalertstate")
         data = {u"tenantId": tenant_id, u"alertsEnabled": alerts_enabled}
         return self._session.post(uri, data=json.dumps(data))
 
-    def get_by_username(self, username, tenant_id=None):
-        """Gets Departing Employee case detail for a given user.
-
-        Arguments:
-            username (str): Username of the Departing Employee to retrieve case info on.
-            tenant_id (str, optional): The identifier of the Customer tenant the case is in.
-                Defaults to None (the tenant_id of SDK authorization user will be used).
-
-        Returns:
-            :class:`py42.sdk.response.Py42Response`: A response containing the Departing Employee case.
-        """
-        tenant_id = tenant_id if tenant_id else self._user_context.get_current_tenant_id()
-        case_id = self._get_case_id_from_username(tenant_id, username)
-        return self.get_by_id(case_id, tenant_id)
-
-    def get_by_id(self, case_id, tenant_id=None):
-        """Gets details about a Departing Employee case.
-        `REST Documentation <https://ecm-default.prod.ffs.us2.code42.com/svc/swagger/index.html#/DepartingEmployeeCase/DepartingEmployeeCase_Details>`__
-
-        Args:
-            case_id (str): Identifier of the Departing Employee case.
-            tenant_id (str, optional): The identifier of the Customer tenant the case is in.
-                Defaults to None (the tenant_id of SDK authorization user will be used).
-
-        Returns:
-            :class:`py42.sdk.response.Py42Response`: A response containing the Departing Employee case.
-        """
-        tenant_id = tenant_id if tenant_id else self._user_context.get_current_tenant_id()
-        uri = self._uri_prefix.format(u"details")
-        data = {u"tenantId": tenant_id, u"caseId": case_id}
-        return self._session.post(uri, data=json.dumps(data))
-
     def update(
```

Comment:
let's call this `update_departure_date`.

Reply:
Changed.